### ISSUE TITLE
Can you tell me md5 or SHA1 of KeePassDroid 2.0.6.1 ?

### ISSUE THREAD

**Issue by wolfsino** [bpellin/keepassdroid#172]
In China we can't download app in google play. Instead, I download KeePassDroid 2.0.6.1 in others source. Today the system says the signature of app seems not the same as official. It is terrible for me. Can you tell me md5 or SHA1 of KeePassDroid 2.0.6.1 so I can check it? Thanks!

**Comment by maikalms** [comment]
hi, for the version on apk pure

md5sum KeePassDroid_v2.0.6.4_apkpure.com.apk
eab27a7054b3a4188a010817214f79cf  KeePassDroid_v2.0.6.4_apkpure.com.apk

sha1sum KeePassDroid_v2.0.6.4_apkpure.com.apk
b4876ecc05609ab6fc833c20b25ec36120bf60db  KeePassDroid_v2.0.6.4_apkpure.com.apk

**Comment by wolfsino** [comment]
Yes, it is the same as mine. Is it official? Thanks.

**Comment by maikalms** [comment]
No, it's from apkpure, I don't know how to get an old version from the official play store, but I think they have unmodified versions. I could compile the old code, but I'm not sure if this will produce the exact same binary an therefore the same hashes. If you want to be a 100 percent sure, wait for an answer from @bpellin
